pkg/email/smtp.go

```diff
@@ -31,62 +30,60 @@ var _ Provider = (*SMTPProvider)(nil)
 type SMTPProvider struct {
 	FirebaseAuth *auth.Client
 
+	Renderer *render.Renderer
+
 	User     string
 	Password string
 	SMTPHost string
 	SMTPPort string
 }
 
 // NewSMTP creates a new Smtp email sender with the given auth.
-func NewSMTP(ctx context.Context, user, password, host, port string, auth *auth.Client) (Provider, error) {
+func NewSMTP(ctx context.Context, user, password, host, port string, h *render.Renderer, auth *auth.Client) Provider {
 	return &SMTPProvider{
 		FirebaseAuth: auth,
+		Renderer:     h,
 		User:         user,
 		Password:     password,
 		SMTPHost:     host,
 		SMTPPort:     port,
-	}, nil
+	}
 }
 
 // SendNewUserInvitation sends a password reset email to the user.
 func (s *SMTPProvider) SendNewUserInvitation(ctx context.Context, toEmail string) error {
```

Comment:
See my note in the other PR, but I think this interface is too coupled to the SMTP provider. SMTP should _only_ know how to send mail.